### DISCUSSION TITLE
What method will you use to display high-order finite-element functions in paraview?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#819] [Q&A] [answered]
I saw a [method](https://github.com/StarGate01/paraview-dev) recently but haven't tried it yet, would like to ask you
What method will you use to display high-order finite-element functions in paraview?

**Comment by krenzland** [comment, marked as answer]
Probably none, as long as there is no support in the standard version of Paraview/vtk.
Right now we use refinement, e.g. as documented here https://seissol.readthedocs.io/en/latest/fault-output.html#refinement

**Reply by buzhiqiuyuebai** [reply]
> Probably none, as long as there is no support in the standard version of Paraview/vtk. Right now we use refinement, e.g. as documented here https://seissol.readthedocs.io/en/latest/fault-output.html#refinement

Will using refinement_strategy significantly reduce the calculation speed? Or just as a post-processing for paraview display?

**Reply by krenzland** [reply]
No, not really. Unless you write output too often. You might need to be a bit careful, as file sizes can become large very quickly.

**Reply by buzhiqiuyuebai** [reply]
> No, not really. Unless you write output too often. You might need to be a bit careful, as file sizes can become large very quickly.

Does it just support Fault output? Is it impossible to apply it to the point source such as WP2-LOH3? I have try to add
```
 &Elementwise
printIntervalCriterion = 2 ! 1=iteration, 2=time
printtimeinterval_sec = 1.0
OutputMask = 1 1 1 1 1 1 1 1 1 1 1 1 !described herafter
refinement_strategy = 1 ! or 2
refinement = 1
/
```
into WP2_LOH3's parameters.par, it doesn't seem to work.

**Reply by krenzland** [reply]
Please refer to the other sections in our documentation which describe all output writers that SeisSol supports.

**Reply by buzhiqiuyuebai** [reply]
> Please refer to the other sections in our documentation which describe all output writers that SeisSol supports.

Thanks!
